data/tlg0593/1st1K001/tlg0593.1st1K001.1st1K-grc1.xml

```diff
@@ -4,10 +4,9 @@
   <teiHeader xml:lang="eng">    
     <fileDesc>
       <titleStmt>
-        <title>Encomium of Helen</title>
         <title xml:lang="grc">ΕΛΕΝΗΣ ΕΓΚΩΜΙΟΝ</title>
```

Comment:
Encomium of Helen (Ἑλένης ἐγκώμιον), Defense of Palamedes (Ὑπέρ Παλαμήδους ἀπολογία), On Non-Existence (Περὶ τοῦ μὴ ὄντος ἢ Περὶ φύσεως), and Epitaphios (Επιτάφιος)

Reply:
Thanks! I've pulled your changes into my branch! Is there anything else I need to do @lcerrato

Reply:
@AlisonBabeu 
Just check my work. Did we catch all of them?